src/Contract/Test/Plutip.purs

```diff
@@ -0,0 +1,18 @@
+-- \ This module contains everything needed for `Contract` testing in Plutip
```

Comment:
```suggestion
-- | This module contains everything needed for `Contract` testing in Plutip
```

Reply:
Haha, my editor renders them almost the same, because it italicizes ordinary comments and not doc comments, so the backslash becomes a vertical line in italic.